CHANGELOG.md

```diff
@@ -5,6 +5,81 @@ Documentation of all notable changes to the **Fizzy** project.
 The format is based on [Keep a Changelog],
 and this project adheres to [Semantic Versioning].
 
+## [0.6.0] — Unreleased
+
+### Added
+
+- `fizzy-wasi` command-line tool to execute Wasm code using WASI interface 
```

Comment:
Mention that WASI support is incomplete, but can be easily extended.

Reply:
Added, but maybe better to mention it above in the description of release.